src/test/java/test/driver/TestLogic.java

```diff
@@ -1,73 +1,95 @@
 package test.driver;
 
-import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
-import org.openqa.selenium.support.ui.ExpectedConditions;
-import org.openqa.selenium.support.ui.WebDriverWait;
 import org.testng.Assert;
 
+
+/**
+ * The class Test logic.
+ */
 public class TestLogic extends PageObject {
 
+    /**
+     * Instantiates a new Test logic.
+     *
+     * @param driver the driver.
+     */
     public TestLogic(WebDriver driver) {
         super(driver);
     }
 
-    public WebDriverWait webDriverWait;
-
-    @FindBy(xpath = "//*[@id='ires' and contains(.,'Selenium - Web Browser Automation')]")
-//    @FindBy(xpath = "//a[contains(@href,'www.seleniumhq.org')]")
-    private WebElement seleniumName;
-
-    @FindBy(xpath = "//a[contains(.,'www.seleniumhq.org')]/span[contains(.,'Selenium')]")
-    //--- $$("div.irc_hd span:nth-child(1)") -- $$("span.oedJWe span:nth-child(1)") --$x("//ancestor::img[parent::id='search'][1]") --$$("div#search a.rg_l>img:nth-of-type(1)")
-//    @FindBy(css = "div.rc a[href='https://www.seleniumhq.org/']")
-//    @FindBy(xpath = "//a[contains(@href,'www.seleniumhq.org')]")
-    private WebElement seleniumNameWhenImage;
-
+    /**
+     * The FindBy searchResult.
+     */
+    @FindBy(css = "div[id='search']")
+    private WebElement searchResult;
+
+    /**
+     * The FindBy imagesTab.
+     */
+    @FindBy(css = "div[id='hdtb-msb-vis'] div:nth-child(2) a")
+    private WebElement imagesTab;
+
+    /**
+     * The FindBy searchField.
+     */
     @FindBy(name = "q")
-    private WebElement findByName;
-
+    private WebElement searchField;
+
+    /**
+     * The FindBy imageElement.
+     */
+    @FindBy(css = "div[class='irc_it'] a[dir='ltr']")
+    private WebElement imageElement;
+
+    /**
+     * The FindBy listOfImages.
+     */
+    @FindBy(css = "#rg_s > div:nth-child(2)")
+    private WebElement listOfImages;
```

Comment:
@vaskocuturilo When I ran the test I got 'NoSuchElementException: Unable to locate element: {"method":"css selector","selector":"#rg_s > div:nth-child(2)"}'. 
Please take a look.